sdk/couchbase-core/src/agent.rs

```diff
@@ -528,13 +530,12 @@ impl Agent {
     fn gen_first_kv_client_configs(
         memd_addrs: &Vec<String>,
         state: &AgentState,
-    ) -> HashMap<String, KvClientConfig> {
+    ) ->HashMap<String, KvClientConfig> {
```

Comment:
Nit: Looks like some spacing was lost.
I'm honestly quite surprised there's not something like rustfmt being used in this codebase

Reply:
I thought we had it running rustfmt checking on PR, will file an issue to add.